evaluate/metrics/metrics_test.go

```diff
@@ -33,7 +77,6 @@ func TestFormatStringCSV(t *testing.T) {
 		},
 
 		ExpectedString: `
-			model,files-total,files-executed,files-problems,coverage-statement,no-excess-response
```

Comment:
Please do the header again. Better to test the whole export. can then be automated as well.